### PR TITLE
MONIT 32316 process cloud watch logs

### PR DESCRIPTION
**Pull request by deosu**
Update the proxy to read logs with a format(logs_json_cloudwatch).
This code change has a dependency on the next java-lib release(which should be released soon). If the latest java-lib release is not `2023-01.1`, then we need to get java-lb changes locally from the feature branch and build the java-lib jar. Then build the proxy jar with the following command. 
`mvn clean install:install-file -Dfile=<path to your java-lib jar> \
   -DgroupId=com.wavefront \
   -DartifactId=java-lib \
   -Dversion=2023-01.1 \
   -Dpackaging=jar`

### CODE REVIEW

**Review by xuranchen — APPROVED** [review]
LGTM

**Comment by deosu** [comment]
@laullon / @joannak-vmware please provide your review on this PR and help me in merging the changes with the dev branch. It is already reviewed by @xuranchen & is QA-certified. Here is a link to the Jira ticket - [MONIT-32316](https://jira.eng.vmware.com/browse/MONIT-32316)

**Review by laullon — APPROVED** [review]
LGTM

**Comment by laullon** [comment]
We need first to release java-lib with your changes, the you can change the version pon the pom.xml, and then merge

**Comment by joannak-vmware** [comment]
LGTM, Will approve PR after pom file changes.

File: proxy/pom.xml
Comment (joannak-vmware) [inline proxy/pom.xml:389]: Need to wait for Java-lib official release and update java-lib version before merge

Comment (deosu) [reply proxy/pom.xml]: @joannak-vmware I have added the latest java-lib.

**Review by joannak-vmware — APPROVED** [review]
LGTM, and java-lib updated

**Pull request merged**
